exonum/src/messages/signed.rs

```diff
@@ -228,17 +228,26 @@ impl<T> ProtobufConvert for Verified<T>
 where
     T: TryFrom<SignedMessage>,
 {
-    type ProtoStruct = proto::Verified;
+    type ProtoStruct = proto::SignedMessage;
 
     fn to_pb(&self) -> Self::ProtoStruct {
-        let mut verified = Self::ProtoStruct::new();
-        verified.set_raw(self.as_raw().to_pb());
-        verified
+        let mut message = Self::ProtoStruct::new();
+        let signed_message = self.as_raw();
+
+        message.set_payload(signed_message.payload.to_pb());
```

Comment:
Yet another nit: `SignedMessage` already implements `ProtobufConvert`. Can we reuse it here?

Like:

```rust
fn to_pb(&self) -> Self::ProtoStruct {
    self.as_raw().to_pb()
}
```

same for `from_pb`.

Reply:
"An hour in the morning is worth two in the evening" 🙂